drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c

```diff
@@ -88,6 +88,11 @@ static const unsigned int rx3_buf_sz = 1000;
 
 struct wifi_nrf_drv_priv_zep rpu_drv_priv_zep;
 
+void wifi_nrf_get_drv_version(void)
```

Comment:
Return type has to be char *. Also lets move this API to  drivers/wifi/nrf700x/osal/fw_if/umac_if/src/fmac_api.c and call it wifi_nrf_fmac_drv_ver_get

Reply:
I will fix the return type, but the version has to be separate per-each OS, so, it should reside in OS specific not agnostic.